bundles/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/model/DataResponseTransformer.java

```diff
@@ -63,6 +63,8 @@ public Map<Channel, State> transform(DataResponse response) {
             if (channel == null) {
                 // This should not happen but we want to get informed about it
                 logger.warn("Channel not found: {}", channelId);
+            } else if (value == null) {
+                logger.info("null value for channel: {}", channelId);
```

Comment:
Why you want to log this to info?

Reply:
Well this is a little complicated. The documentation regarding which channels exist for which model is not very good. In addition the channels depend not only on the model but also on some extra components. I added this to show it in the logs if a channel does not return any data. Of course you will also see that the item will never show any value but this could also happen in case of a configuration error.

Reply:
In that case i think debug is a better fit.